.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
+known_numpy=numpy
```

Comment:
Is this the only way to acheve that?
What if we set the force_alphabetical_order instead?

Reply:
It would work but it won't separate numpy imports from keras imports

Reply:
We don't want to separate them, they are both third-party libraries.